### PR TITLE
evm: add tx.authorizationList field

### PR DESCRIPTION
**Pull request by tmcgroul**

### CODE REVIEW

File: evm/evm-processor/src/interfaces/evm.ts [hunk 1]
```diff
@@ -35,6 +35,16 @@ export interface EvmTransaction extends _EvmTx, _EvmTxReceipt {
 }
 
 
+export interface Authorization {
```
Comment (tmcgroul) [inline evm/evm-processor/src/interfaces/evm.ts:38]: @eldargab do we need `Evm` prefix as for other entities?

File: evm/evm-processor/src/mapping/schema.ts [hunk 2]
```diff
@@ -64,6 +74,7 @@ export function getTxProps(fields: FieldSelection['transaction'], forArchive: bo
             s: withSentinel('Transaction.s', '0x', BYTES),
             yParity: option(natural),
             chainId: option(natural),
+            authorizationList: option(array(Authorization)),
```
Comment (tmcgroul) [inline evm/evm-processor/src/mapping/schema.ts:77]: @eldargab do we need ability to select each field manually or it's fine and we can load the whole array of structures?

Comment (eldargab) [reply evm/evm-processor/src/mapping/schema.ts:77]: Let's expose it as a whole.

File: evm/evm-processor/src/interfaces/evm.ts (hunk 1 above)
Comment (eldargab) [reply evm/evm-processor/src/interfaces/evm.ts:38]: Prefix might be helpful, but don't forget to re-export `Authorization` in a prefix-less form.

I'm afraid `Authorization` is too ambiguous, though. May be call it like `EIP7702Authorization`?

**Pull request merged**